### DISCUSSION TITLE
Health bars not displaying with some snake colors

### DISCUSSION THREAD

**Discussion by bvanvugt** [BattlesnakeOfficial/feedback#6] [Bug Reports]
**What's the problem?**
In [this game](https://play.battlesnake.com/g/8f80d133-65ad-4125-87b2-018cd94c6563/) Playful Puppy's healthbar is invisible even though the snake renders as black.

**Notes**
This requires a bit of investigation to figure out the root cause.

**Screenshot Showing the Issue**
![Screenshot 2020-03-14 19 48 35](https://user-images.githubusercontent.com/1531419/76694184-fef0a780-662c-11ea-9b78-268c4baa96c5.png)

**Comment by hanneskuettner** [comment]
Without actually looking at it in this specific situation I noticed that the problem probably is that the snake color coming from the engine (and therefore the user snake) is an invalid hex color code. (Missing # for example) and the board defaults to black (default fill) for snakes and white (default background of divs used for healthbar) for the bars.
My suggestion would be to (1) do some validation from the engine side and provide an invalid snake with a nice default color and (2) to fix the board that the default colors are consistent.

**Comment by jstolp** [comment]
Can you favorite the game? Or post the JSON. Its gone...

**Comment by andrewdmaclean** [comment]
@bvanvugt is this still an issue?

**Comment by bvanvugt** [comment]
We don't believe this is an issue in the current board.
